spring-rabbit/src/main/java/org/springframework/amqp/rabbit/connection/ConnectionFactoryUtils.java

```diff
@@ -25,6 +23,8 @@
 import org.springframework.transaction.support.TransactionSynchronizationManager;
 import org.springframework.util.Assert;
 
+import com.rabbitmq.client.Channel;
```

Comment:
I haven't changed anything in my STS; I have always had these settings, which (I think) I got from Mark:

![screen shot 2015-12-08 at 10 12 25 am](https://cloud.githubusercontent.com/assets/483832/11659086/64a24cac-9d94-11e5-9242-89c85996d224.png)